packages/exo_base.rb

```diff
@@ -0,0 +1,50 @@
+require 'package'
+
+class Exo_base < Package
+  description 'Environment set-up for Chrome OS built-in Wayland server'
+  homepage 'https://github.com/skycocker/chromebrew/'
+  version '1.0'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  depends_on 'wayland'
+
+  def self.preflight
+    abort 'Wayland is not supported in your Chrome OS version'.lightred unless File.stat('/var/run/chrome/wayland-0').socket?
+  end
+
+  def self.build
+    @env = <<~EOF
+      # environment set-up for Chrome OS built-in Wayland server
+      set -a
+      # all variables will export automatically
+
+      XDG_RUNTIME_DIR=/var/run/chrome
+      XDG_SESSION_TYPE=wayland
+      WAYLAND_DISPLAY=wayland-0
+
+      CLUTTER_BACKEND=wayland
+      GDK_BACKEND=wayland
+
+      set +a
+    EOF
+  end
+
+  def self.install
+    system "echo '#{@env}' >> #{HOME}/.bashrc" unless `grep -c '#{@env.lines.first.chomp}' #{HOME}/.bashrc`.to_i > 0
```

Comment:
Shouldn't this go in `def self.postinstall` since it's not in the packaged files?

Reply:
Done